### PR TITLE
Auth for comment section

### PR DESCRIPTION
**Pull request by danieltre23**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -22,12 +22,26 @@ function openModal(num) {
...
         start = 0;
         if (commentsArray.length === 0) {
             document.getElementById('commentDependent').setAttribute('class', 'hide');
         }
         assginCommentsToUI();
+
+        if (!data.user) {
+            document.getElementById('commentForm').setAttribute('class', 'hide');
+            document.getElementById('loginBtn').setAttribute('href', data.url);
+            console.log("no user");
```
Comment (wujingya) [inline portfolio/src/main/webapp/script.js]: If this is for debugging purpose, let's remove it before merging

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -22,12 +22,26 @@ function openModal(num) {
...
+        } else {
+            document.getElementById("commentUser").value = data.user;
+            document.getElementById("hiddenUser").value = data.user;
+            document.getElementById('commentForm').setAttribute('class', 'text-center padding-10-px');
+            document.getElementById('loginDiv').setAttribute('class', 'hide');
+            document.getElementById('logoutBtn').setAttribute('href', data.url);
+            console.log(data.user);
```
Comment (wujingya) [inline portfolio/src/main/webapp/script.js]: same here

File: portfolio/src/main/java/com/google/sps/data/ServerResponse.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+import com.google.sps.data.Comment;
+import java.util.ArrayList;
+
+public final class ServerResponse {
```
Comment (wujingya) [inline portfolio/src/main/java/com/google/sps/data/ServerResponse.java:6]: nice!

**Review by wujingya — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (danieltre23) [reply portfolio/src/main/webapp/script.js]: Yes, I forgot to delete them, thanks

**Pull request merged**